### PR TITLE
Add Japanese translations

### PR DESCRIPTION
**Pull request by ykzts**
follow up #3261 and #3634

### CODE REVIEW

File: config/locales/ja.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ ja:
     people_who_follow: "%{name} さんをフォロー中のアカウント"
     posts: 投稿
     remote_follow: リモートフォロー
+    reserved_username: このユーザー名は予約されています。
```
Comment (clworld) [inline config/locales/ja.yml]: It follows up #3566 also.

File: config/locales/ja.yml [hunk 2]
```diff
@@ -226,6 +231,7 @@ ja:
       content: セキュリティ認証に失敗しました。Cookieをブロックしていませんか？
       title: セキュリティ認証に失敗
     '429': リクエストの制限に達しました。
+    noscript: Mastodonを利用する場合はJavaScriptを有効にしてください。
```
Comment (clworld) [inline config/locales/ja.yml]: I wondered which is better '利用' or '使用', but '利用' is good enough.

**Review by clworld — APPROVED** [review]
Looks good to me.

File: config/locales/ja.yml (hunk 2 above)
Comment (unarist) [inline config/locales/ja.yml]: '利用' is good but it might sound like saying about Mastodon as a service, not only our web client. So it may be better with '使用' '使う', or rephrase it to something like:

- JavaScript を有効にしてください
- このページを表示するには JavaScript が必要です

but this is not a huge problem I think.

(Mentioning something like "Instead, you can use Mastodon from 3rd-party apps" may also be good, but this will be no longer translation issue)

Comment (ykzts) [inline config/locales/ja.yml]: Certainly "use" has a slight incongruity. However, I do not want to change it significantly from the original phrase.

The difference between "使用" and "利用" is difficult. However, since it is provided as a web service, I thought that "利用" is more appropriate.
